recipes/graphene/all/conanfile.py

```diff
@@ -25,23 +38,20 @@ class LibnameConan(ConanFile):
         "with_glib": True,
     }
 
-    _source_subfolder = "source_subfolder"
-    _build_subfolder = "build_subfolder"
-
     def config_options(self):
         if self.settings.os == "Windows":
             del self.options.fPIC
         if self.settings.compiler == "gcc":
-            if tools.Version(self.settings.compiler.version) < "5.0":
+            if Version(self.settings.compiler.version) < "5.0":
                 raise ConanInvalidConfiguration("graphene does not support GCC before 5.0")
-    
+
     def build_requirements(self):
-        self.build_requires("meson/0.61.2")
-        self.build_requires("pkgconf/1.7.4")
-    
+        self.tool_requires("meson/0.63.3")
+        self.tool_requires("pkgconf/1.9.3")
```

Comment:
this can be in the conf so we do no force it https://github.com/conan-io/conan-center-index/blob/50a3aed534573d6f1ca8b13195f089dad4f31e83/docs/package_templates/autotools_package/all/conanfile.py#L98